### PR TITLE
add name to ExecutableShimTask, sqlalchemy dockerfile fix

### PR DESCRIPTION
**Pull request by cosmicBboy**
Signed-off-by: Niels Bantilan <niels.bantilan@gmail.com>

# TL;DR
There's an issue with the sqlalchemy flytekitplugin, which leads to a `ModuleNotFoundError` when we try to execute a workflow on `pyflyte run` that uses the `SqlAlchemyTask` (see [execution](https://demo.nuclyde.io/console/projects/flytesnacks/domains/development/executions/f46a67e59cb4d4d5f861?duration=all)).

This PR installs `flytekitplugins-sqlalchemy` to the sqlalchemy docker image and adds a `name` property to the `ExecuteableShimTask`, where the cli `entrypoint.py` complains that the name attribute is not defined (see [execution](https://demo.nuclyde.io/console/projects/flytesnacks/domains/development/executions/f65f24c3ffc674c25adb?duration=all))

## Type
 - [X] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Review by wild-endeavor — DISMISSED** [review]
let me know after you back out the test changes.

File: .github/workflows/sqlachemy_publish.yml [hunk 1]
```diff
@@ -30,6 +33,6 @@ jobs:
           push_git_tag: true
           push_image_and_stages: true
           registry: ghcr.io
-          build_extra_args: "--compress=true --build-arg=tag=ghcr.io/${{ github.repository_owner }}/flytekit:sqlalchemy-${{ github.sha }}"
+          build_extra_args: "--compress=true --build-arg=VERSION=${{ needs.deploy.outputs.version }} --build-arg=tag=ghcr.io/${{ github.repository_owner }}/flytekit:sqlalchemy-${{ github.sha }}"
```
Comment (wild-endeavor) [inline .github/workflows/sqlachemy_publish.yml:36]: wait we're still only doing the sha for the tag?

**Review by wild-endeavor — DISMISSED** [review]

File: plugins/flytekit-sqlalchemy/Dockerfile.py3.10 [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+RUN apt-get update -y
+RUN apt-get install git -y
+
+RUN pip install awscli
+RUN pip install gsutil
+
+ARG VERSION
+
+# Virtual environment
+RUN python3.8 -m venv ${VENV}
```
Comment (eapolinario) [inline plugins/flytekit-sqlalchemy/Dockerfile.py3.10:18]: these should match the versions from the base image?

Comment (cosmicBboy) [reply plugins/flytekit-sqlalchemy/Dockerfile.py3.10:18]: good catch @eapolinario !

**Review by wild-endeavor — APPROVED** [review]
